lib/util/tooltip.coffee

```diff
@@ -57,17 +83,21 @@ class Tooltip
     hideTimer = null
     showTimer = null
     @parent.onmouseover = =>
-      @positionOverlay()
       clearTimeout hideTimer
-      if @cond() then showTimer = setTimeout (=> @show()), @delay
+      if @cond() then showTimer = setTimeout (=> @show()), @showDelay
     @parent.onmouseout = =>
       clearTimeout showTimer
-      hideTimer = setTimeout (=> @hide()), @delay
-    @view.onmouseover  = => clearTimeout hideTimer
-    @view.onmouseout   = => hideTimer = setTimeout (=> @hide()), @delay
+      hideTimer = setTimeout (=> @hide()), @hideDelay
+    @view.onmouseover  = =>
+      clearTimeout hideTimer
+    @view.onmouseout   = =>
+      hideTimer = setTimeout (=> @hide()), @hideDelay
 
   positionOverlay: ->
     bounding = @parent.getBoundingClientRect()
-    @view.style.bottom   = bounding.height + 'px'
-    @view.style.left     = bounding.left   + 'px'
-    @view.style.minWidth = bounding.width  + 'px'
+    @view.style.bottom = document.documentElement.clientHeight - bounding.top + 'px'
+    @view.style.left = switch @position
+      when 'left'
+        bounding.left + 'px'
+      when 'right'
+        bounding.left + bounding.width - @view.offsetWidth + 'px'
```

Comment:
Yeah, that was (kinda) by design. Before I implemented [this](https://github.com/JunoLab/atom-ink/pull/127/files#diff-720cdfa58eff6266473cf704034d6dd0R95) toolbars would look strange on small results.
Will fix this though.